overpass/api.py

```diff
@@ -204,9 +207,19 @@ def search(self, feature_type, regex=False):
         """Search for something."""
         raise NotImplementedError()
 
+    def __deprecation_get(self, *args, **kwargs):
```

Comment:
Everything from this point, in this file, is still relevant with the dropping of 3.6.